### PR TITLE
Switching channels

### PR DESCRIPTION
**Pull request by monikasudol**
Switching channels, please look at it and review

### CODE REVIEW

File: src/state/login/index.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { handleActions, createAction } from 'redux-actions';
 
 const initialState = ({
-  loading: false
+  loading: false,
+  invalidLogin: 'false'
```
Comment (yuriyyakym) [inline src/state/login/index.js]: why string?

File: src/modules/main/navigator/channels/channel/index.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
 import './style.css';
+import { clearMessageBoard, fetchMessages } from '../../../../../state/messages';
 
-class Channel extends Component {
+export class Channel extends PureComponent {
 
   handleDirectToChannel = () => {
-    console.log('to do');
+    const { id } = this.props;
+    this.props.onClearMessageBoard();
```
Comment (yuriyyakym) [inline src/modules/main/navigator/channels/channel/index.js]: Do it in saga

File: src/state/messages/index.js [hunk 3]
```diff
@@ -23,9 +22,10 @@ export const messagesReducer = handleActions({
     messages: [{}]
   }),
 
-  [FETCH_MESSAGES]: (state) => ({
+  [FETCH_MESSAGES]: (state, { payload }) => ({
     ...state,
-    loading: true
+    loading: true,
+    id: payload.id
```
Comment (yuriyyakym) [inline src/state/messages/index.js]: id -> conversationId

File: src/state/login/index.js (hunk 1 above)
Comment (monikasudol) [reply src/state/login/index.js]: You haven't seen that.. i was checking sth while doing test.. nevermind :D

**Pull request merged**